### ISSUE TITLE
Fix clone directions for nomal users

### ISSUE THREAD

**Issue by HorstBaerbel** [sciplot/sciplot.github.io#1]
`git clone https://github.com/sciplot/sciplot --recursive` should be `git clone https://github.com/sciplot/sciplot` for regular users

**Comment by allanleal** [comment]
Yes, you're right. The git submodules are needed only for documentation generation and updating the `Palettes.hpp` file.
